apps/explorer/src/components/transaction-card/TxCardUtils.tsx

```diff
@@ -101,22 +100,10 @@ export function TxAddresses({ content }: { content: LinkObj[] }) {
 }
 
 function TxStatusType({ content }: { content: TxStatus }) {
```

Comment:
Is this wrapper component now necessary? Why not just remove it and use the new component instead?